EIPS/eip-894.md

```diff
@@ -0,0 +1,39 @@
+## Preamble
+
+    EIP: 894
+    Title: Immutability Enforcement Proposal (IMP)
+    Author: Marius Kjærstad
+    Type: Meta
+    Status: Draft
+    Created: 2018-02-21
+
+
+## Definition
+The definition of bailouts in this EIP is state changes to the Ethereum blockchain, through a hardfork, that treat account holders in different ways.
+
+## Simple Summary
+Provide a standardized response against proposals for bailouts on the offical Ethereum repositories.
+
+
+## Abstract
+Proposals that advocate for bailouts on the Ethereum blockchain should not be merged into the official Ethereum repositories.
+
+
+## Motivation
+The issue of bailouts on the Ethereum blockchain is always controversial. This EIP attempts to raise the barriers for bailouts.
+
+
+## Specification
+If a pull request is opened that advocates for bailouts or implements code that will execute bailouts on the Ethereum blockchain, it will be closed by the maintainers.
+
+
+## Justification
+Bailouts play favorites with some accounts on the Ethereum blockchain. This is corruption we can not tolerate in the offical Ethereum repositories.
```

Comment:
That's your **opinion**

Reply:
@AtLeastSignificant you are absolutely right. I just believe, that `justification` should contain something more than just an opinion e.g.
- irregular state changes is likely to cause a network split, like it happened with the dao fork
- if a network split happens, we are likely to cause X ether to be lost due to transaction duplication
- my twitter poll shows that 20 % of users will move to ethereum classic, cause it's immutable :trollface: 

just anything

Reply:
So following basic ethics is not a justification?

Reply:
What do "basic ethics" have to do with this?  What basic ethical argument are you actually making?

Reply:
Personally I have a "basic ethical" principle that redefining words to mean things most people don't understand them to mean in order to push your political agenda is unethical.

Reply:
Bailout is the most fitting word I can think of.